requirements.txt

```diff
@@ -1,6 +1 @@
-mlflow[extras]==1.17.0
-
-# MLFlow's 
-psycopg2-binary==2.8.6
-pymysql==1.0.2
-mysqlclient==2.0.3
```

Comment:
What if someone uses postres or mysql in the action config? Will it work?

Reply:
Good catch! Added back in 99565a1.
Also, for the future, asked in MLFlow maintainers regarding those deps, m.b. we will have some reasonable info https://github.com/mlflow/mlflow/issues/4878